python/PIK_maya_loader/scripts/PIKmenu/loader.py

```diff
@@ -0,0 +1,108 @@
+import  sys
+import  json
+from    maya    import cmds
+from    maya    import mel
+from    maya    import OpenMaya as om
+
+class MenuLoader:
+
+    def __init__(self, menuName):
+        """Init the menu
+        """
+        self._menuName = menuName
+        self.gMainWindow = mel.eval( '$tmpVar = $gMainWindow' )
+
+
+    def createMenu(self):
+        """Create the main menu
+
+        :return: The created menu
+        :rtype: object cmds.menu
+        """
+        om.MGlobal.displayInfo("# PIK_maya_loader - loading menu")
+
+        # Check if the menu exists, delete if it does
+        if (cmds.menu(self._menuName, exists = True)):
+            cmds.deleteUI(self._menuName)
+        
+        # Create the menu
+        menu = cmds.menu(self._menuName, parent=self.gMainWindow)
+        cmds.menuItem(divider = True, dividerLabel = 'PIKTURA - tools', parent = menu)
+
+        self.createSubMenusWithScripts(menu)
+
+
+    def createSubMenusWithScripts(self, parent):
+        """Create a deployable menu item for each sub menu
+
+        :param parent: The main menu
+        :type parent: Maya menu object
+        """
+
+        # Get sub menus
+        folderPath, loaderFile = os.path.split(os.path.normpath(__file__))
+        subMenus = [
+            folder for folder in os.listdir(folderPath)
+            if os.path.isdir(os.path.join(folderPath, folder))
+            and not "__pycache__" in folder
+        ]
+
+        for sm in subMenus:
+            # Create sub menus
+            subMenu = cmds.menuItem(sm, label = sm, subMenu = True, parent = parent)
+            subMenuScripts = os.listdir(os.path.join(folderPath, sm))
+
+            for script in subMenuScripts:
+                om.MGlobal.displayInfo("# PIK_maya_loader - loading %s"%script)
+
+                # Get script datas               
+                datas = os.path.join(folderPath, sm, script, 'datas.json')
+
+                if not os.path.isfile(datas):
+                    continue
+                with open(datas, 'r') as jsonFile:
+                    scriptDatas = json.load(jsonFile)
+
+                # Get script content
+                with open(os.path.join(folderPath, sm, script, 'main.py'), 'r') as s:
+                    scriptContent = s.read()
+
+                # Create menu item
+                cmds.menuItem(scriptDatas['scriptName'],
+                    annotation = scriptDatas['annotation'],
+                    image = "%s.png"%script,
+                    command = scriptContent,
+                    sourceType = scriptDatas['scriptType'],
+                    parent = subMenu)
+
+        # Add an item to reload the menu #TODO
+        #cmds.menuItem(divider = True, parent = self._menuName)
+        #cmds.menuItem(label = 'Refresh menu', command = self.refreshMenu ,parent = self._menuName)
+                
+
+    def refreshMenu(self, *args):
+        """Delete the menu and reload it
+
+        :raises Exception: userSetup.py wasn't found
```

Comment:
Please use Google Style for docstring: https://google.github.io/styleguide/pyguide.html#244-decision
```suggestion
        Raises:
            Exception: `userSetup.py` not found.
```